### PR TITLE
[stable/envoy] Add support for static load balancer IP: bugfix

### PR DESCRIPTION
**Pull request by khorshuheng**
Signed-off-by: Shu Heng <solemnpriest@gmail.com>

#### What this PR does / why we need it:
This is to fix the bug introduced in #18219, which has the wrong attribute name.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:
I apologize for not noticing this earlier when i submitted the PR.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/envoy/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Envoy is an open source edge and service proxy, designed for cloud-native applications.
 name: envoy
-version: 1.8.0
```
Comment (mumoshu) [inline stable/envoy/Chart.yaml]: @khorshuheng Hey! Thanks for the fix. Could you rebase onto the latest `master` and change the version number here to `1.8.2`?

Comment (khorshuheng) [reply stable/envoy/Chart.yaml]: I have rebased and update the version number.

Comment (mumoshu) [reply stable/envoy/Chart.yaml]: @khorshuheng LGTMed. Thanks for your effort 👍

**Pull request merged**